UWP/UWP.vcxproj.filters

```diff
@@ -110,6 +110,88 @@
       <Filter>Content</Filter>
     </None>
     <None Include="PPSSPP_UWP_TemporaryKey.pfx" />
+<<<<<<< HEAD

```

Comment:
Diff markers left behind, this won't compile.

Reply:
Damn, I didn't catch that since for some reason my code compiles fine. I'll check as soon as I get home and fix it. Is there are another observation or does the other changes check out and you're ok with them for now?

Reply:
Fixed this on latest commit